pkgs/development/node-packages/node-env.nix

```diff
@@ -363,7 +354,7 @@ let
 
         npm ${forceOfflineFlag} --nodedir=${nodeSources} ${npmFlags} ${stdenv.lib.optionalString production "--production"} rebuild
 
-        if [ "''${dontNpmInstall-}" != "1" ]
+        if [ "$dontNpmInstall" != "1" ]
```

Comment:
This should stay!

Reply:
I have no recollection from this change. Most likely the generated data from 1.6.1. You mean this should stay, the change or the original? (I assume the original, in which case regenerating with right version should fix it).

Reply:
It is a bug in node2nix itself: https://github.com/svanderburg/node2nix/pull/174

Reply:
We need to keep the current line.